src/app/app-routing.module.ts

```diff
@@ -21,6 +20,7 @@ const routes: Routes = [
 
 @NgModule({
   imports: [RouterModule.forRoot(routes)],
-  exports: [RouterModule]
+  exports: [RouterModule],
+  providers: [AuthGuard]
```

Comment:
I removed the guard from provider, but in order to avoid error I needed to add in the guard: ```
@Injectable({
  **providedIn: 'root'**
})
``` Is it OK?